.github/workflows/release.yaml

```diff
@@ -3,7 +3,8 @@ name: release
 on:
   push:
     tags:
-      - '[0-9]+.[0-9]+.[0-9]+'
+      - v*.*.*
+      - v*.*.*-rc.*
```

Comment:
Did you mean to restore the `v` prefix for version tags?

Reply:
Yes. The tag needs the v (all our repos have v in the version tags), is just the application version (published) that removes the v. This is something I've addressed in the kotlin-sdk repo itself when the tag is received